src/azure/azureController.ts

```diff
@@ -211,4 +208,19 @@ export class AzureController {
         await azureAuth.deleteAccountCache(account.key);
         return;
     }
+
+    /**
+     * Checks if this.init() has already been called
+     */
+    private async initializeHelper(): Promise<void> {
```

Comment:
Suggest just calling this `initializeCredentialStore` or something similar to make it clear what it's doing